### ISSUE TITLE
Integration with XNAT

### ISSUE THREAD

**Issue by jhegman** [rii-mango/Papaya#156]
Has anyone integrated this with the XNAT imaging platform? Currently I have all my imaging stored in XNAT. I am using XNAT's API tp request an image and then pass a base64 encoded string to Papaya. This is working ok, but worried about performance as some files are 100mb large.

Is there a way to read in an image by passing a remote url, or does the file have to be saved locally?

Any ideas would be appreciated.

**Comment by martinezmj** [comment]
Yes, I integrated it with XNAT in the past.  I don't remember the specifics, but I was able to get DICOM URLs from XNAT and then passed them to a page that loaded Papaya.  (I remember there was some kind of plugin architecture in XNAT that I used to integrate Papaya.)  

You definitely don't need to do anything with base64 encoded data for this, just pass Papaya the URLs to the DICOMs: https://github.com/rii-mango/Papaya/wiki/Configuration#images

```
params["images"] = [["/path/to/mySeries/Image1", "/path/to/mySeries/Image2"]];
```

The encoded data mode is only for the special case of wanting to work with local files.
